packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/Data.tsx

```diff
@@ -0,0 +1,64 @@
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+import { Paragraph, Row } from '@trezor/components';
+import { palette, spacingsPx } from '@trezor/theme';
+import styled from 'styled-components';
+import { Translation } from 'src/components/suite';
+import { TranslationKey } from '@suite-common/intl-types';
+
+const Container = styled.div`
```

Comment:
You might use `Grid` component there.

Reply:
how do I do 
```
    grid-template-columns: 130px 3fr;
```

There is this in the `Grid`
```
    grid-template-columns: repeat(${({ $columns }) => $columns}, 1fr);
```

Reply:
So maybe not 😂 let's skip it